### ISSUE TITLE
Identify docket links on any website to any website and add [R] links

### ISSUE THREAD

**Issue by mlissner** [freelawproject/recap#358]
We've been timid on this front, but I think it's reasonable to be a bit more aggressive, if we can. We should start identifying links to other websites that host dockets, and start injecting our little [R] icons next to those links, so people can open our docket pages, if the other one isn't great, requires a login, whatever. 

This can live behind a permissions prompt, and can be an option in the settings that is defaulted to off, but I think the idea is useful and sound. The permissions prompt would expand our permissions boundary to include all domains, not just RECAP ones.

**Comment by mlissner** [comment]
I think an example would help here. Imagine that you're a pro-openness attorney that's at a corporation. The corp is subscribed to Docket Bird/Alarm/PacerPro/etc, and you get an email from them for whatever reason, because their service is great and the folks that decide on tools picked that one for you.

In the email, there's a link to the docket on their website. What we can do is put our link next to it, so people can look up the case on CourtListener, contribute to it there, etc.

I'm also open to doing a bit more than just an [R] icon. It could have a dropdown with additional features like "search this docket", "buy on PACER", etc, though this is getting harder!

**Comment by arcataroger** [comment]
> The permissions prompt would expand our permissions boundary to include all domains, not just RECAP ones.

Just a thought: Can we limit it to known domains/URLs that we want to want to inject into, instead of requesting "all domains"? E.g. something like this:

https://github.com/IsThereAnyDeal/AugmentedSteam/blob/c5f4cd5396bdb92230a9ecda5902f7716fc1e7a2/config/manifests/base.json#L49-L374

Which then shows up to the user like:
![image](https://github.com/freelawproject/recap/assets/11964962/e1afcc21-25b9-48a1-ac78-01d88524cd5d)

This is less scary, I think, than "This extension can view and change your data on all websites".

We should also investigate whether it's possible to delay those permissions until the user specifically turns on the feature for those specific sites. (I'm not sure if the Chrome extension permission model allows that, but I can check)

**Comment by mlissner** [comment]
Yeah, I think putting the permissions prompt behind a setting would make this much more palatable, but I don't know that we can come up with a complete list of domains where we'd want to look for links to dockets.

**Comment by mlissner** [comment]
Just got a request to add PACER Monitor scraping to the extension.
